src/textual_summary/tag_group.jsx

```diff
@@ -6,30 +6,52 @@ import IconOrImage from './icon_or_image';
  * Render a group of tags in a table.
  */
 export default class TagGroup extends React.Component {
+  /**
+   * Return onClick function if link is defined for an item/subitem
+   */
+  checkLinkItem = (item, e) => {
+    if (item.link && this.props.onClick) {
+      this.props.onClick(item, e);
+    }
+  }
+
   /**
    * Render a simple row. Label, icon, value.
    */
   renderTagRowSimple = item => (
     <tr key={item.key}>
       <td className="label">{item.label}</td>
-      <td>
-        <IconOrImage icon={item.icon} image={item.image} title={item.title} />
-        {item.value}
+      <td title={item.title} onClick={e => this.checkLinkItem(item, e)}>
+        <IconOrImage icon={item.icon} image={item.image} title={item.title} /> {item.value}
```

Comment:
keep the 2 lines as 2 lines please :)

Reply:
Do you think it is better to add `&nbsp;` before `{item.value}` if I want to achieve a nice space between icon and the item name? Like this:
![icon_space](https://user-images.githubusercontent.com/13417815/60901474-52288400-a26e-11e9-867f-17f983df5691.png)
Because I think we don't want this:
![icon_close](https://user-images.githubusercontent.com/13417815/60901490-5785ce80-a26e-11e9-9a88-8afd48a59eaa.png)

Moving ``{item.value}` to the same line as `IconOrImage` was the simplest and working fix for this.

Reply:
Ah, sorry, react is not html...

I think the canonical way to do this (if I understand https://github.com/facebook/jsx/issues/19#issuecomment-57079949 right) is..

```jsx
<td title={item.title} onClick={e => this.checkLinkItem(item, e)}>
  <IconOrImage icon={item.icon} image={item.image} title={item.title} />
  {' '}
  {item.value}
</td>
```